pkg/sql/catalog/internal/catkv/stored_catalog.go

```diff
@@ -124,6 +161,38 @@ func (sc *StoredCatalog) ensure(ctx context.Context, desc catalog.Descriptor) er
 	return nil
 }
 
+func (sc *StoredCatalog) ensureComment(
+	ctx context.Context, key catalogkeys.CommentKey, cmt string,
+) error {
+	if existing, ok := sc.comments[key]; ok {
+		sc.memAcc.Shrink(ctx, int64(len(existing)))
+	}
+	sc.comments[key] = cmt
+	if err := sc.memAcc.Grow(ctx, int64(len(cmt))); err != nil {
+		return err
+	}
+	return nil
+}
+
+// markZoneConfigInfoSeen see storedZoneConfigs.markSeen
```

Comment:
nit: This comment reads strangely, not sure it's useful

Reply:
agreed, removed.